src/pages/Merchant/Register/Register.js

```diff
@@ -0,0 +1,180 @@
+        dummyErrors.password = "Must contain at least one letter";
+        setErrors(dummyErrors);
+      }
+      if (passRef.current.value.search(/[0-9]/) < 0) {
+        dummyErrors.password = "Must contain at least one digit";
+        setErrors(dummyErrors);
+      }
+    }
+
+    if (confirmpassRef.current.value === "") {
+      dummyErrors.confirmpass = "Confirm password";
+      setErrors(dummyErrors);
+    } else {
+      if (confirmpassRef.current.value !== passRef.current.value) {
+        dummyErrors.confirmpass = "passwords must be same";
+        setErrors(dummyErrors);
+      }
+    }
+  };
+
+  const handleSubmit = (event) => {
+    event.preventDefault();
+    validateForm();
+  };
+
+  const navigate = useNavigate();
+  const changeURl = () => {
+    navigate("/login");
+  };
+
+  return (
+    <div className={styles.register}>
+      <div className={styles.registerLeftPanel}>
+        <h1>Buy Best!</h1>
+        <span>100+ products available at best price</span>
+        <img
+          className={styles.registerLeftPanelImage}
+          src="https://image.shutterstock.com/z/stock-vector-set-of-fruit-and-vegetables-logo-for-groceries-agriculture-stores-packaging-and-advertising-318421853.jpg"
+        ></img>
+        <div className={styles.registerLeftPanelIcons}></div>
+      </div>
+      <div className={styles.registerRightPanel}>
+        <form onSubmit={handleSubmit}>
+          <div className={styles["registerRightPanel-mainBody"]}>
+            <h2>Create an account</h2>
+            <div className={styles["registerRightPanel-inputContainer"]}>
+              <div className={styles["registerRightPanel-inputWrapper"]}>
+                <InputControl
+                  placeholder="Enter first name"
+                  label="First name"
+                  ref={fnameRef}
+                  error={errors?.fname}
+                />
+              </div>
+              <div className={styles["registerRightPanel-inputWrapper"]}>
+                <InputControl
+                  label="Last name"
+                  placeholder="Enter last name"
+                  ref={lnameRef}
+                  error={errors?.lname}
+                />
+              </div>
+            </div>
+            <div className={styles["registerRightPanel-inputWrapper"]}>
+              <InputControl
+                placeholder="Enter mobile number"
+                label="Mobile number"
+                ref={mobileRef}
+                error={errors?.mobile}
+              />
+            </div>
+            <div className={styles["registerRightPanel-inputWrapper"]}>
+              <InputControl
+                label="Email"
+                placeholder="Enter email"
+                ref={emailRef}
+                error={errors?.email}
+              />
+            </div>
+            <div className={styles["registerRightPanel-inputContainer"]}>
+              <div className={styles["registerRightPanel-inputWrapper"]}>
```

Comment:
same inputWrapper mistake as in login page

Reply:
done